aws-arn.cabal

```diff
@@ -51,11 +51,11 @@ common opts
 
 common deps
   build-depends:
-    , base             >=4.12   && <4.18
+    , base             >=4.12   && <4.19
     , deriving-compat  >=0.5.10 && <0.7
     , profunctors      >=5.0    && <5.7
     , tagged           ^>=0.8
-    , text             ^>=1.2.3 || ^>=2.0
+    , text             ^>=1.2.3 || >=2.0 && <2.1 || ^>=2.1
```

Comment:
Why this and not `^>=1.2.3 || >=2.0 && <2.2` ?

Reply:
Beats me, but that's what `cabal-fmt` turned it into.